mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -91,15 +97,15 @@ func resourceMongoDBAtlasProjectCreate(ctx context.Context, d *schema.ResourceDa
 
 	project, _, err := conn.Projects.Create(ctx, projectReq, createProjectOptions)
 	if err != nil {
-		return diag.FromErr(fmt.Errorf(errorProjectCreate, err))
```

Comment:
This is not in the PR description, can you describe?  Is this updates for the newer SDK that weren't made yet?

Reply:
I was working on this file and I noticed this, the correct use is `diag.Errorf(.....)` not `diag.FromErr(fmt.Errorf(....))` (this way works but is not correct)

Reply:
Glad to have it corrected but next time this kind of things come up please describe it and why in the PR description.  That way it's not a surprise. Thank you.